Ease.Unity/UnityContainerTestBase.cs

```diff
@@ -1,35 +1,35 @@
-﻿using System;
-using Moq;
+﻿using Moq;
+using System;
 using Unity;
-using Unity.Injection;
 using Unity.Lifetime;
-using Unity.Registration;
 
-namespace Ease.NUnit.Unity
+namespace Ease.Unity
 {
-	public class UnityContainerTestBase : ContainerTestBase
+	public abstract class UnityContainerTestBase : ContainerTestBase
 	{
-		protected UnityContainer Container;
-		protected LifetimeResetter Resetter { get; set; }
+		protected UnityContainer _container;
```

Comment:
see comments on `DryIocContainerTestBase` re naming and calling the reset through a protected method on test framework class. Also I see I made these protected, but should be private.

Reply:
Done